src/wayfinding/types/index.ts

```diff
@@ -61,3 +68,16 @@ export type NavigateToPoiType = {
   navigationTo: number;
   type?: string;
 };
+
+export type NavigationAccessibilityTypes =
```

Comment:
Change to enum

Reply:
export enum RouteType {
  CHOOSE_SHORTEST = "CHOOSE_SHORTEST",
  ONLY_ACCESSIBLE = "ONLY_ACCESSIBLE",
  ONLY_NOT_ACCESSIBLE_FLOOR_CHANGES = "ONLY_NOT_ACCESSIBLE_FLOOR_CHANGES",
}